pkg/awsclient/client.go

```diff
@@ -66,6 +66,9 @@ type Client interface {
 	DeleteUser(*iam.DeleteUserInput) (*iam.DeleteUserOutput, error)
 	DeleteUserPolicy(*iam.DeleteUserPolicyInput) (*iam.DeleteUserPolicyOutput, error)
 	GetUser(*iam.GetUserInput) (*iam.GetUserOutput, error)
+	ListUsers(*iam.ListUsersInput) (*iam.ListUsersOutput, error)
+	ListUsersPages(*iam.ListUsersInput, func(*iam.ListUsersOutput, bool) bool) error
```

Comment:
Where is this function defined? I'm not seeing it in the SDK but I maybe be blind.

Reply:
https://docs.aws.amazon.com/sdk-for-go/api/service/iam/#IAM.ListUsersPages